lib/briefly/entry.ex

```diff
@@ -114,6 +115,7 @@ defmodule Briefly.Entry do
     IO.iodata_to_binary([tmp, "/",
                          prefix(options),
                          "-", i(sec), "-", i(micro), "-", i(scheduler_id),
+                         "-", i(:rand.uniform(10000)),
```

Comment:
This isn't a guarantee either, seems like we need something like `:erlang.unique_integer`